openwisp_ipam/tests/test_api.py

```diff
@@ -251,6 +251,24 @@ def test_hosts_list_api(self):
         self.assertEqual(self.client.get(response.data['previous']).status_code, 200)
         self.assertIsNone(response.data['next'])
 
+    def test_bearer_auth(self):
+        self._create_user(username='tester', password='tester')
+        params = {'username': 'tester', 'password': 'tester'}
+        url = reverse('users:user_auth_token')
+        r = self.client.post(url, params)
+        token = r.data
```

Comment:
data is a dict, so to get the token, you will need to do `token=r.data['token']`